_pages/plugins/ultrack.md

```diff
@@ -0,0 +1,109 @@
+Please refer to the [Ultrack documentation](https://github.com/royerlab/ultrack) for the installation instructions.
+This plugin will call the Ultrack routines under the hood, and the user will be able to track cells in 2D+t and 3D+t 
+datasets.
+
+### FIJI integration
+
+The ultrack plugin is available after selecting the update site for ultrack in FIJI by clicking **HELP** → **Update** 
+→ **Manage Update Sites** and then searching for **Ultrack**. To activate the update site, click the checkbox in the
+**Activate** column, the leftmost one. Right after that, click **Apply and Close** and then **Apply Changes** in the now
+enabled button at the bottom of the previous **ImageJ Update** window. That will trigger the download of the Ultrack
+plugin. 
+
+After that, FIJI restart is required and then the Ultrack plugin will be available in the **Plugins**→**Tracking**→
+**Ultrack** menu.
+
+## How to Use
+
+The usage is simple and the following steps describes briefly how to use the Ultrack plugin:
+
+1. Open the image sequence to be tracked.
+
+2. Click on the **Plugins** → **Tracking** → **Ultrack** menu. The Ultrack GUI will open.
+
+[![Ultrack GUI](/media/plugins/ultrack/00_init.png)](/media/plugins/ultrack/00_init.png)
+
+3. (Optional but required in the first run) If ultrack is correctly installed and accessible through the system path, 
+   the user can proceed to the next step. Otherwise, the user needs to set the path to the ultrack executable in the 
+   **Environment**→**Select Conda Path** menu. Firstly, it will open a dialog to select the `conda` executable and 
+   then the user needs to select the desired environment, which should be one indicated with the `[Ultrack found]` flag.
+
+4. Now you should be able to start the connection with the Ultrack software by clicking the **Start Ultrack Server** 
+   button. The plugin will update its GUI to show the available tracking options, as shown in the figure below.
+
+[![Ultrack GUI with server started](/media/plugins/ultrack/01_gui.png)](/media/plugins/ultrack/01_gui.png)
+
+4. The user now should select the desired tracking workflow in this section:
+    
+[![Ultrack GUI with tracking workflows](/media/plugins/ultrack/03_workflows.png)](/media/plugins/ultrack/03_workflows.png)
+    
+in which the user can select the desired tracking workflow. The options are:
+ - **Auto Detection From Image**: which provides a way to track cells using solely the image as input;
+ - **Manual Detection**: which is suitable for tracking cells using your custom **edge** detection and **foreground** 
+   segmentation algorithms;
+ - and **Auto Detection From Label**: which is suitable for tracking cells using a pre-segmented image from your favorite 
+   segmentation software, such as [Stardist](https://github.com/stardist/stardist), 
+   [Cellpose](https://github.com/MouseLand/cellpose), 
+   [MicroSAM](https://github.com/computational-cell-analytics/micro-sam) or any other.
+
```

Comment:
Do you think `Foreground & edges from user` and `Segmentation from user` might be clearer for items 2 and 3? I'll check Xiang and Merlin for this. It might need a refactor in the UI.